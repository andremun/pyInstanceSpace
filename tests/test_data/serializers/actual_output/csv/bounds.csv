Row,z_1,z_2
bnd_pnt_1,2.850254254723516,-3.23945833000166
bnd_pnt_2,3.9024444390972097,-2.23610038815957
bnd_pnt_3,4.333153900695586,-0.8450118850401171
bnd_pnt_4,4.63973337318227,0.5050352790331045
bnd_pnt_5,4.621248894168246,3.5821143927293964
bnd_pnt_6,3.8883182162047563,4.107570037134741
bnd_pnt_7,2.88300429705912,4.4331092575451745
bnd_pnt_8,-0.623646711250949,5.026973914577049
bnd_pnt_9,-1.546205314328072,4.781291357901016
bnd_pnt_10,-2.0951724431376406,4.481635929469855
bnd_pnt_11,-3.0461578523577244,3.629015120849348
bnd_pnt_12,-4.098348036731418,2.625657179007258
bnd_pnt_13,-4.529057498329795,1.2345686758878054
bnd_pnt_14,-4.835636970816478,-0.11547848818541645
bnd_pnt_15,-4.817152491802454,-3.1925576018817083
bnd_pnt_16,-4.084221813838964,-3.7180132462870525
bnd_pnt_17,-3.078907894693328,-4.043552466697486
bnd_pnt_18,0.42774311361674117,-4.63741712372936
bnd_pnt_19,1.3503017166938642,-4.391734567053327
bnd_pnt_20,1.8992688455034326,-4.092079138622166
bnd_pnt_21,2.850254254723516,-3.23945833000166
