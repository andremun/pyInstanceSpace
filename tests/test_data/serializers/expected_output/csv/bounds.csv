Row,z_1,z_2
bnd_pnt_1,2.85025425472352,-3.23945833000166
bnd_pnt_2,3.90244443909721,-2.23610038815957
bnd_pnt_3,4.33315390069559,-0.845011885040117
bnd_pnt_4,4.63973337318227,0.505035279033105
bnd_pnt_5,4.62124889416825,3.5821143927294
bnd_pnt_6,3.88831821620476,4.10757003713474
bnd_pnt_7,2.88300429705912,4.43310925754517
bnd_pnt_8,-0.623646711250949,5.02697391457705
bnd_pnt_9,-1.54620531432807,4.78129135790102
bnd_pnt_10,-2.09517244313764,4.48163592946985
bnd_pnt_11,-3.04615785235772,3.62901512084935
bnd_pnt_12,-4.09834803673142,2.62565717900726
bnd_pnt_13,-4.5290574983298,1.23456867588781
bnd_pnt_14,-4.83563697081648,-0.115478488185416
bnd_pnt_15,-4.81715249180245,-3.19255760188171
bnd_pnt_16,-4.08422181383896,-3.71801324628705
bnd_pnt_17,-3.07890789469333,-4.04355246669749
bnd_pnt_18,0.427743113616741,-4.63741712372936
bnd_pnt_19,1.35030171669386,-4.39173456705333
bnd_pnt_20,1.89926884550343,-4.09207913862217
bnd_pnt_21,2.85025425472352,-3.23945833000166
